marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -302,15 +324,15 @@ handleQuery (ReceiveWebSocketMessage msg next) = do
                                 -- TODO: SCP-3487 swap store contract from new to running
                                 addToast $ successToast
                                   "Contract initialised."
-                          Just (EndpointSuccess _ ApplyInputsResponse) ->
+                          EndpointSuccess _ ApplyInputsResponse ->
```

Comment:
Have you tested this part? I think the `Just` was there because of a serialization issue

Reply:
I can't test it, since I'm not getting updates from the PAB. I think it's OK to test once that is working, and fix if need be, WDYT?

Reply:
No, it was to represent if `onNewObservableState` found a pending result. This is still handled, implicitly, via `traverse_` above.